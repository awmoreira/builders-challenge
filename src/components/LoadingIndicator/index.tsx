import React from 'react';
import { ActivityIndicator, ActivityIndicatorProps } from 'react-native';
import Colors from '~/global/colors';

const LoadingIndicator: React.FC<ActivityIndicatorProps> = ({ size = 'small', color = Colors.white }) => (
  <ActivityIndicator color={color} size={size} />
);

export default LoadingIndicator;
