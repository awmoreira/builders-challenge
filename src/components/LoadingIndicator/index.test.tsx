import React from 'react';
import renderer from 'react-test-renderer';
import LoadingIndicator from '.';

describe('Loading Indicator Component', () => {
  it('should render component', () => {
    const tree = renderer.create(<LoadingIndicator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
