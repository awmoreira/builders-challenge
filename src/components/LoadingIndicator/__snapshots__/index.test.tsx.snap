// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Indicator Component should render component 1`] = `
<ActivityIndicator
  color="#fff"
  size="small"
/>
`;
